open_humans/tests.py

```diff
@@ -10,13 +10,13 @@
 UserModel = auth.get_user_model()
 
 ANONYMOUS_URLS = [
+    '/',
     '/account/login/',
     '/account/password/reset/',
     '/account/signup/',
 ]
 
 AUTHENTICATED_OR_ANONYMOUS_URLS = [
-    '/',
```

Comment:
@madprime `/welcome/profile/` should be added to `AUTHENTICATED_URLS` so it can be tested :)